### ISSUE TITLE
No root request

### ISSUE THREAD

**Issue by CapitalH** [ElderDrivers/EdXposed#158]
Hi, I posted this on the Xda tread and was told to put it here.

So, just an observation that I wonder if it has been over looked.

I notice in the change log for the xposed manager here: https://github.com/solohsu/XposedInstaller/releases that they got rid of ROOT permission at v2.2.0

I have found that using edx 0.2.2.9b2/riru 15 (most reliable for me currently)and xposed manager 4.2.1 everything ONLY works properly once I grant the manager root access. If I don't give root permission, I get framework installed and active but no modules work.

If I then try using the latest xposed manager v2.2.3 in the above link, (without root permission request) WITH edx 0.2.2.9b2/riru 15, then the xposed framework only ever says installed and active but none of the modules work. 

This is the same behaviour I get when I use the latest edx, riru and manager 2.2.3

I'm wondering if getting rid of root permission is causing issues? Thanks

**Comment by solohsu** [comment]
I can't figure out what was going on from what you said. Please install latest versions and post your logs here.

**Comment by CapitalH** [comment]
Hi

I've installed latest riru, edx and xinstaller 2.2.3.

This time I was able to get root permission prompt for xinstaller which helped but everything only works when I soft reboot from inside installer. 

If I reboot from the Android power menu, it always results in Xinstaller showing xposed not installed (red)

Here's a link for two logs: one after power menu reboot resulting in "xposed not installed". 

And and one after soft reboot from inside Xinstaller resulting in everything working.

https://drive.google.com/file/d/191_rtjMKDXL9syH8cHeHTMkqz451PSw0/view?usp=drivesdk

**Comment by CapitalH** [comment]
Hi

So, in regards to my previous post with logs attached, I think I made a mistake with having the latest Riru installed. The version I had installed was v17 when the actual latest was v17.1

Sorry about that.

I have however installed the latest updates you've just released including: edxp 0.3.1.7 and edxp installer 2.2.4

The first reboot straight after updating, framework was active and xposed modules were working, however, everytime now when I reboot, xposed framework is active but no modules work anymore! Even trying soft reboot doesn't work. 

Attached as the logs after a reboot

https://drive.google.com/file/d/1YhE9AJGO718r3H9WdzOV4EXQ8LdD99vU/view?usp=drivesdk

**Comment by solohsu** [comment]
Need system boot log.

**Comment by CapitalH** [comment]
Not sure how to get that and not sure which instructions to follow to get what you need, sorry.

**Comment by solohsu** [comment]
@CapitalH 
Use the latest Log-Catcher module in below comment to get boot logs.
https://github.com/ElderDrivers/EdXposed/issues/131#issuecomment-473502478
module page:
https://github.com/MlgmXyysd/Log-Catcher

**Comment by CapitalH** [comment]
Hi, sorry for such a delayed reply.

So I've just updated to latest everything:

Riru v18
Edxposed v0.4.1.2_beta(YAHFA) 
Edxposed Manager v4.3.2
Magisk v18.1
Magisk manager v7.1.1

After install then reboot, xposed framework was RED and "not installed". 

I finally got root access prompt by using reboot from inside xposed manager but only after several attempts.

Now after a normal system reboot Xposed Framework is GREEN and active, modules are activated after another reboot but no xposed modules work. 

Here is a boot log:

https://drive.google.com/file/d/15OF7wl9M4jn3xHc7xH5z10SWBZLXUAlh/view?usp=drivesdk
